assets/lang/pl_PL.ini

```diff
@@ -225,14 +225,14 @@ Allow remote debugger = Zezwól na zdalne debugowanie
 Backspace = Backspace
 Block address = Adres bloku
 By Address = Po adresie
-Copy savestates to memstick root = Copy save states to Memory Stick root
+Copy savestates to memstick root = Skopjuj zapisane stany do folderu głównego Karty Pamięci
 Create/Open textures.ini file for current game = Stwórz/otwórz plik textures.ini z bieżącej gry
 Current = Obecny
 Dev Tools = Narzędzia Developerskie
 DevMenu = DevMenu
 Disabled JIT functionality = Wyłącz funkcje JIT
 Draw Frametimes Graph = Draw frametimes graph
-Dump Decrypted Eboot = Zachowuj odszyfrowany EBOOT.BIN przy starcie
+Dump Decrypted Eboot = Zachowuj odszyfrowany EBOOT.BIN przy starcieSkopjuj zapisane stany do folderu głównego Karty Pamięci
```

Comment:
problem z Ctrl+C, Ctrl+V?

Reply:
nie wiem co to biło tbh :smile:
używam vim'a więc Ctrl+C/V raczej nie wchodzi w grę
nie mniej dzięki ;-) 9007504